modules/update-other-images-40.adoc

```diff
@@ -79,6 +79,11 @@ Or if you're using collector slim:
 ----
 $ oc -n stackrox set image ds/collector collector=registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:{rhacs-version}
 ----
+. Update the Admission Control image:
++
+----
+$ oc -n stackrox set image deploy/admission-control admission-control=registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version} <1>
```

Comment:
![image](https://user-images.githubusercontent.com/537715/151779517-ef5bc598-b2bd-4075-87b9-f627cf8897bd.png)
```suggestion
$ oc -n stackrox set image deploy/admission-control admission-control=registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version}
```

Reply:
Again 😵‍💫 
Sorry about that